tests/unittests/Foundation/NSCharacterSetTests.m

```diff
@@ -219,22 +219,7 @@ void testCharacter(NSCharacterSet* charSet, NSString* testString) {
 
     set = [NSMutableCharacterSet characterSetWithRange:NSMakeRange(0, 32)];
     EXPECT_TRUE([set isKindOfClass:[NSMutableCharacterSet class]]);
-}
-
-ARM_DISABLED_TEST(NSCharacterSet, ExceptionTests) {
-    NSMutableCharacterSet* set;
-    @try {
-        set = [NSMutableCharacterSet characterSetWithBitmapRepresentation:nil];
-        FAIL();
-    } @catch (NSException* e) {
-        ASSERT_OBJCEQ(e.name, @"NSInvalidArgumentException");
-    }
 
-    @
-    try {
-        set = [NSMutableCharacterSet characterSetWithContentsOfFile:nil];

```

Comment:
It merely calls characterSetWithBitmap but with the contents of the file. It would merely be testing file lookup. As characterSetWithBitmap isn't tested either, there's no point in exploring this function before it gets any actual attention. A real test needs to be developed for this function which is not the priority of this pull request.